scripts/run_e2e_tests.py

```diff
@@ -467,6 +470,8 @@ def main(args=None):
     if not parsed_args.skip_build:
         build_js_files(
             dev_mode, deparallelize_terser=parsed_args.deparallelize_terser)
+    else:
```

Comment:
Should we change this to:
```
if skip_buld:

...
else:
...
```

Also, can we use `parsed_args.prod_env` instead of `not dev_mode`?

Reply:
Yep, done.